docs/t8ntool-ref.rst

```diff
@@ -0,0 +1,298 @@
+.. _t8ntool_ref:
+
+###########################################
+Transition Tool
+###########################################
+
+
+
+Command Line Parameters
+=======================
+The command line parameters are used to specify the parameters, input files, and
+output files.
+
+Test Parameters
+---------------
+- **-\\-state.fork** *fork name*
+- **-\\-state.reward** *block mining reward* (appears only in Block tests)
+- **-\\-trace** produce a trace
+
+Input Files
+-----------
+- **-\\-input.alloc** *full path to pretest allocation file*
+- **-\\-input.txs** *full path to transaction file*
+- **-\\-input.env** *full path to environment file*
```

Comment:
It's either the full path to, but any field can be set to `stdin` if user wants to pass that data as stdin (see my gist)

Reply:
Are there any users of `t8ntool` other than `retesteth`? If not, why do we even have `stdin` and `stdout`? They only complicate the implementation for client developers.

Reply:
The main idea, I guess, is to allow execution that doesn't touch disk, so e.g. retesteth wouldn't have to actually dump things to disk just to run a testcase, but just feed everything via the proc stdin. 
I didn't know how @winsvega would implement it. 
stdin is faster, but files are more convenient if (as I am mostly) you just run a few tests and want it reproducible.